.lando.yml

```diff
@@ -82,6 +87,20 @@ services:
         PHP_IDE_CONFIG: serverName=appserver
         # PHPUnit settings. @see: .lando/phpunit.sh
         MINK_DRIVER_ARGS_WEBDRIVER: '["chrome", {"browserName":"chrome","chromeOptions":{"args":["--disable-gpu","--headless"]}}, "http://chrome:9515"]'
+  node:
+    type: node:16
+    ssl: true
```

Comment:
this is needed to have the https://frontend.lndo.site using https, which is needed because otherwise the browser does not allow to have a http iframe inside an https site.

Reply:
(this means they will need to accept the warning, or trust lando's ca authority)